doc/manual/manual.tex

```diff
@@ -1315,14 +1316,233 @@ \subsection{Numerical methods}
 
 \subsection{Simplifications of the basic equations}
 
-There are two common variations to equations
-\eqref{eq:stokes-1}--\eqref{eq:temperature} that are frequently used and that
-make the system much simpler to solve and analyze: assuming that the fluid is
-incompressible (the Boussinesq approximation) and a linear dependence of the
-density on the temperature with constants that are otherwise independent of
-the solution variables. These are
-discussed in the following; \aspect{} has
-run-time parameters that allow both of these simpler models to be used.
+There are a number of common variations to equations
+\eqref{eq:stokes-1}--\eqref{eq:temperature} that are frequently used in the
+geosciences. For example, one frequently finds references to the anelastic liquid
+approximation (ALA), truncated anelastic liquid approximation (TALA), and the
+Boussinesq approximation (BA). These can all be derived from the basic
+equations~\eqref{eq:stokes-1}--\eqref{eq:temperature} via various approximation, though
+there are a number of pitfalls if one wanted to use them in \aspect{}. We will
+discuss these in the following. Since they are typically only provided
+considering velocity, pressure and temperature, we will in the following omit
+the dependence on the compositional fields used in previous sections, though
+this dependence can easily be added back into the equations stated below. A
+detailed discussion of the approximations introduced below can also be found in
+\cite{KLKLZTTK10}.
+
+The three approximations mentioned all start by writing the pressure and
+temperature as the sum of a (possibly depth dependent)
+reference value plus a perturbation, i.e., we will write
+\begin{align*}
+  p(\mathbf x,t) &= \bar p(z) + p'(\mathbf x,t),
+  \\
+  T(\mathbf x,t) &= \bar T(z) + T'(\mathbf x,t).
+\end{align*}
+Here, barred quantities are reference states and may depend on the depth $z$
+(not necessarily the third component of $\mathbf x$) whereas primed quantities
+are the spatially and temporally variable deviations of the temperature and
+pressure fields from this reference state. In particular, the reference pressure
+is given by solving the hydrostatic equation,
+\begin{align*}
+  \nabla \bar p = \bar\rho \mathbf g,
+\end{align*}
+where $\bar\rho=\rho(\bar p,\bar T)$ is a \textit{reference density} that
+depends on depth. $\bar T$ is chosen as an adiabatic profile accounting for the
+fact that the temperature increases as the pressure increases, but also taking
+\marginpar{Scott: I don't see this discussed in \cite{KLKLZTTK10}, but one
+needs to make $\bar T$ also take into account diffusion so that it disappears
+from the temperature equation in the ALA.} 
+into account thermal diffusion effects. It also depends on depth.
```

Comment:
Do you mean to say that it *explicitly* depends on the depth? If yes, I would say "explicitly", otherwise, I would say "Consequently, through the dependence on the pressure, it varies with depth."